examples/react/kitchen-sink/src/index.css

```
@@ -289,6 +289,47 @@ html {
+.cell-selectable {
+  cursor: cell;
+  user-select: none;
+  --cell-edge-top: 0 0 0 0 transparent;
+  --cell-edge-right: 0 0 0 0 transparent;
+  --cell-edge-bottom: 0 0 0 0 transparent;
+  --cell-edge-left: 0 0 0 0 transparent;
+  box-shadow:
+    inset var(--cell-edge-top),
+    inset var(--cell-edge-right),
+    inset var(--cell-edge-bottom),
+    inset var(--cell-edge-left);
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)